src/NorthwindWebsite/Configuration/ServicesConfiguration.cs

```diff
@@ -46,10 +52,62 @@ public static class ServicesConfiguration
         services.AddScoped<ICategoryRepository, CategoryRepository>();
         services.AddScoped<ISupplierRepository, SupplierRepository>();
 
+        services.AddScoped<IEmailSender, EmailSender>();
```

Comment:
Probably Singleton scope will suit here better because you will not have to create HTTP-Client for object per each request, it will reduce cost amounts of server performance